packages/main/src/Menu.ts

```diff
@@ -528,23 +516,24 @@ class Menu extends UI5Element {
 		}
 		if (item && item.hasSubmenu) {
 			// create new sub-menu
-			this._createSubMenu(item, actionId);
+			this._createSubMenu(item, opener, actionId);
 			this._openItemSubMenu(item, opener, actionId);
 		}
 		if (this._parentMenuItem) {
 			this._parentMenuItem._preventSubMenuClose = true;
 		}
 	}
 
-	_prepareSubMenuPhone(item: MenuItem) {
-		this._prepareCurrentItems(item.items);
-		this._parentMenuItem = item;
-		this._parentItemsStack.push(item);
+	_onfocusin(e: CustomEvent<any>): void {
+		const target = e.target as HTMLElement;
+		const opener = target instanceof MenuListItem ? target : (target.getRootNode() as any).host;
```

Comment:
Minor: as this is focus event, in my opinion "opener" is not so proper, maybe "menuListItem" is better?